### PR TITLE
Add force_update option to RemoteFileSet

### PR DESCRIPTION
**Pull request by ronisbr**
Hi @helgee !

Sorry, I forgot to add the option `force_update` to the `RemoteFileSet`. This problem is fixed in this PR.

### CODE REVIEW

File: src/RemoteFiles.jl [hunk 1]
```diff
@@ -319,11 +319,13 @@ Download all files contained in `rfs`.
 - `quiet`: Do not print messages.
 - `verbose`: Print all messages.
 - `force`: Force download and overwrite existing files.
+- `force_update`: Overwrite existing files even if they are equal.
 """
-function download(rfs::RemoteFileSet; quiet::Bool=false, verbose::Bool=false, force::Bool=false)
+function download(rfs::RemoteFileSet; quiet::Bool=false, verbose::Bool=false, force::Bool=false,
```
Comment (helgee) [inline src/RemoteFiles.jl]:
```suggestion
function download(rfs::RemoteFileSet; kwargs...)
```

File: src/RemoteFiles.jl [hunk 2]
```diff
@@ -319,11 +319,13 @@ Download all files contained in `rfs`.
...
+    force_update::Bool=false)
```
Comment (helgee) [inline src/RemoteFiles.jl]:
```suggestion
```

File: src/RemoteFiles.jl [hunk 3]
```diff
@@ -319,11 +319,13 @@ Download all files contained in `rfs`.
...
     verbose && @info "Downloading file set '$(rfs.name)'."
     @sync for file in values(rfs.files)
-        @async download(file, quiet=verbose, verbose=verbose, force=force)
+        @async download(file, quiet=verbose, verbose=verbose, force=force, force_update=force_update)
```
Comment (helgee) [inline src/RemoteFiles.jl]:
```suggestion
        @async download(file; kwargs...)
```

**Review by helgee — COMMENTED** [review]
Since all keyword arguments are being passed through to the file `download` call, this would be much simpler, right?

**Comment by ronisbr** [comment]
> Since all keyword arguments are being passed through to the file `download` call, this would be much simpler, right?

Yes! It should be better!

**Comment by helgee** [comment]
All except `verbose` 🙈 But that should be fixed now...

**Pull request merged**